src/app/components/SelectOption/index.jsx

```diff
@@ -0,0 +1,54 @@
+import { Select } from 'antd';
+import React, { useState } from 'react';
+
+import './styles.css';
+
+const { Option } = Select;
+
+export default function SelectOption({
```

Comment:
i will change this component name to `SelectOptionCustom`
what's do you think with that name?

Reply:
A `Select` will contains its Option(s). So that the name `SelectOption` can make other devs confused. Your suggested names: `SelectOption` and `SelectOptionCustom` are pointing to the same thing: the option element/component of your custom `Select`.